docs/get-started/installing-cardano-node.md

```diff
@@ -515,6 +515,16 @@ cp -p "$(./scripts/bin-path.sh cardano-node)" $HOME/.local/bin/
 cp -p "$(./scripts/bin-path.sh cardano-cli)" $HOME/.local/bin/
 ```
 
+In order to run the cardano-node as a service using systemd you need to run the following command:
```

Comment:
Since there are heaps of Linux flavours around:

Simply running ldconfig may not be enough (neither absence of it will guarantee the error, if  LD_LIBRARY_PATH already points to local lib folder).

If using ldconfig, it would be an essential step to ensure /usr/local/lib - or the path where you install the lib to - is part of /etc/ld.so.config (or mentioned in a file in /etc/ld.so.conf.d).